src/multistore.rs

```diff
@@ -0,0 +1,131 @@
+// mostly a copy of https://github.com/filecoin-project/merkle_light/blob/master/src/store/vec.rs
+
+use std::ops::Range;
+
+use anyhow::Result;
+
+use merkletree::merkle::Element;
+use merkletree::store::{DiskStore, Store, StoreConfig, VecStore};
+
+#[derive(Debug)]
+pub struct MultiStore<E: Element>(DiskStore<E>, VecStore<E>);
```

Comment:
I would make this a struct with fields:

```rust
pub struct MultiStore<E: Element> {
    disk: DiskStore<E>,
    mem: VecStore<E>
};
```

This way you can access them with `self.disk` and `self.mem` instead of `self.0` and `self.1` which is easier to read and less error-prone.

Reply:
nice!